summingbird-storm/src/main/scala/com/twitter/summingbird/storm/builder/BaseBolt.scala

```diff
@@ -79,6 +81,17 @@ case class BaseBolt[I, O](jobID: JobId,
   private[this] val rampPeriods = maxExecutePerSec.rampUptimeMS / PERIOD_LENGTH_MS
   private[this] val deltaPerPeriod: Long = if (rampPeriods > 0) (upperBound - lowerBound) / rampPeriods else 0
 
+  private[this] val outputFields = outputEdges.headOption.map(_.edgeType.fields).getOrElse(new Fields())
+  assert(
+    outputEdges.forall(_.edgeType.fields.toList == outputFields.toList),
+    s"$boltId: Output edges should have same `Fields` $outputEdges")
+  private[this] val outputInjection = outputEdges.headOption.map(_.edgeType.injection).orNull
```

Comment:
It was for a case with empty output edges, which was handled later. I change type to `Option[Injection[[]]`.